extensions/darwin/clock-nanosleep.c

```diff
@@ -0,0 +1,80 @@
+/* Copyright (C) 2020-2022 Aleix Conchillo Flaqué <aconchillo@gmail.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation; either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301 USA
+ */
+
+#include "clock-nanosleep.h"
+
+#include <errno.h>
+#include <time.h>
+#include <sys/types.h>
+
+/**
+ * The code below has been extracted from the following notcurses files:
```

Comment:
In that case, the copyright information appears to be incorrect.

Also, the Apache license (which notcurses uses) requires

4. Redistribution. You may reproduce and distribute copies of the Work or Derivative Works thereof in any medium, with or without modifications, and in Source or Object form, provided that You meet the following conditions:

    You must give any other recipients of the Work or Derivative Works *a copy of this License*; and
    You must cause any modified files to carry *prominent notices stating that You changed the files*; and
    [....]

(emphasis added) which hasn't been done here.

Reply:
I just replaced LGPL license with Apache licenese and removed my Copyright line. I also left the old comment. Please, let me know if this is now OK.

Reply:
Resolving. Unresolve if I didn't make it right.